OrbitClientServices/CrashManager.cpp

```diff
@@ -28,16 +30,19 @@ void CrashManagerImpl::CrashOrbitService(
   CrashOrbitServiceRequest request;
   request.set_crash_type(crash_type);
 
-  CrashOrbitServiceResponse response;
   grpc::ClientContext context;
+  std::chrono::time_point deadline =
+      std::chrono::system_clock::now() +
+      std::chrono::milliseconds(kTimeoutMilliseconds);
+  context.set_deadline(deadline);
+
+  CrashOrbitServiceResponse response;
+  grpc::Status status =
+      crash_service_->CrashOrbitService(&context, request, &response);
 
-  std::function<void(grpc::Status)> callback = [](::grpc::Status status) {
-    if (!status.ok()) {
-      ERROR("Grpc call failed: %s", status.error_message());
-    }
-  };
-  crash_service_->experimental_async()->CrashOrbitService(&context, &request,
-                                                          &response, callback);
+  if (!status.ok()) {
+    ERROR("Grpc call failed: %s", status.error_message());
```

Comment:
we kind of expect it to fail with deadline aren't we? maybe check if it returns expected deadline exceeded error?

Reply:
Yes, you're right: I added a check for this, thanks!